src/sx/src/__tests__/__snapshots__/fixtures.test.js.snap

```diff
@@ -36,8 +36,7 @@ exports[`matches expected output: @supports.js 1`] = `
     color: #f00;
   }
 }
-@supports (transform-style: preserve) or (-moz-transform-style: preserve) or
-  (-o-transform-style: preserve) or (-webkit-transform-style: preserve) {
```

Comment:
So it also removes unneeded prefixes? Cool! 🙂

This makes me think though: shouldn't we specify some `.browserslistrc` explicitly for these tests? Which versions are they using now? Isn't it a bit flaky now?

Reply:
We could but it would probably have to be in the root folder. 

Or is there a way how we can pass env vars to some tests? https://github.com/browserslist/browserslist#environment-variables

Reply:
> Or is there a way how we can pass env vars to some tests?

You can use https://jestjs.io/docs/en/tutorial-react-native#setupfiles to set env vars for test files.

Reply:
Looks like `process.env.BROWSERSLIST = 'last 2 versions';` in `jest.config.js` did that job